prometheus/counter.go

```diff
@@ -78,6 +89,9 @@ type counter struct {
 	desc *Desc
 
 	labelPairs []*dto.LabelPair
+	exemplar   atomic.Value // *dto.Exemplar
```

Comment:
full stop

Reply:
Hmm, that looks weird as this comment is not in English but in Go.

To resolve the ambiguity here, I changed it to actual English: `Always a *dto.Exemplar.`

Reply:
Or more precisely: `Containing nil or a *dto.Exemplar.`